apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql

```diff
@@ -376,6 +389,7 @@ type Query {
   userAuditLogs(id: UUID!): [UserAuditedChangeAuditLogDto!]!
   userRoleById(id: UUID!): UserRoleWithPermissionsDto!
   userProfile: GetUserProfileResponse!
+  userById(id: UUID!): GetUserResponse!
```

Comment:
nitpick: I know you are just using the dto from markpart which is totally fine, but maybe we should consider creating our own `User` type, so it looks like this `userById(id: UUID!): User!`. And then we could even consider using that for the `UserOverviewItemDto` as well, since they look almost the same. But doesn't have to be in this PR.

Reply:
Good idea, will do it in other PR